ui/src/App.scss

```diff
@@ -213,3 +213,32 @@ body {
 .ant-notification-topRight {
   z-index: $zindex-header + 5 !important;
 }
+
+.ant-page-header.site-page-header.ant-page-header-ghost {
+  padding: 4px 0;
+  .ant-page-header-heading-title {
+    font-weight: 500;
+    font-size: 18px;
+    padding-right: 5px;
+  }
+}
+
+.ant-btn.btn-ghost {
+  border: none;
+  box-shadow: none;
+  height: 26px;
+  padding: 0 5px;
+  margin: 4px 10px;
+  font-size: 12px;
+  border-radius: 0;
+  color: rgba(94, 94, 94, 0.5);
+  background-color: #f0f2f5;
+  border: none;
+  &:hover {
+    border: none;
+    box-shadow: none;
+    color: rgba(94, 94, 94, 0.6);
+    background-color: #e0e1e3;
+    border: none;
+  }
+}
```

Comment:
what is that for? maybe you can add css for that in the same dir as the component/container that's using it?